src/explore-education-statistics-common/src/components/ContentSectionIndex.module.scss

```diff
@@ -0,0 +1,3 @@
+.link {
+  word-break: break-word;
```

Comment:
Maybe it might be better if we just do this for all link stlyes in `_typography.scss`?

Reply:
I'm a bit wary of making a global change in case there's unexpected consequences, but I'll try it locally and have a look around the site to see if it causes any problems.

Reply:
Seems fine so have changed it, good call :)